nms/app/packages/magmalte/app/views/network/FEGNetworkInfo.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {DataRows} from '../../components/DataGrid';
+import type {feg_network} from '@fbcnms/magma-api';
+
+import DataGrid from '../../components/DataGrid';
+import React from 'react';
+
+type Props = {
+  fegNetwork: $Shape<feg_network>,
+};
+
+/**
+ * Returns information about the federation network.
+ * @param {object} props: has a property called fegNetwork that has
+ * information about the federation network.
+ */
+export default function NetworkInfo(props: Props) {
+  const kpiData: DataRows[] = [
+    [
+      {
+        category: 'ID',
+        value: props.fegNetwork.id,
+      },
+    ],
+    [
+      {
+        category: 'Name',
+        value: props.fegNetwork.name,
+      },
+    ],
+    [
+      {
+        category: 'Description',
+        value: props.fegNetwork.description || '-',
+      },
+    ],
+    [
+      {
+        category: 'Federation',
+        value: props.fegNetwork.id,
+      },
+    ],
+  ];
+  return <DataGrid data={kpiData} testID="feg_info" />;
+}
```

Comment:
another item that it would be interesting to show is inside `federation` key

```
    "served_network_ids": [
      "terravm2_inbound_agw_network"
    ],
    "served_nh_ids": [
      "terravm2_feg_network"
    ],
```

`served_network_ids` indicates the LTE federated networks this federation serves.

`served_nh_ids` indicates the Neutral Host (or Virtual) Fedration Networks it serves

Both are array of strings.

Reply:
Added them.
![Screen Shot 2021-06-25 at 9 46 41 AM](https://user-images.githubusercontent.com/34602743/123435022-18f09800-d59b-11eb-8d18-77f3c40ec881.png)

Reply:
@uri200 Can you suggest right hints to be added with each of these items. It will be extremely confusing for an average user to understand what served federated LTE network ID is and served virtual federation network ID

Reply:
Hey @karthiksubraveti, I don't think the user should configure gateways based on the hints. The hints are useful to show the format they need to input but not the concept. The users will have to check the guide first to see if those fields are even necessary. 

For example `served_nh_ids` is used on Inbound Roaming. Once we have final field I will add a note to the instructions to use that.

`served_network_ids`  is the ide of the federated LTE networks this FEG serves


Note both fields are an array, so it would be good to display the format the user needs to use to add more than one

Reply:
> Hey @karthiksubraveti, I don't think the user should configure gateways based on the hints. The hints are useful to find the field and show the format they need to input but they should not used as a definition. The users will have to check the guide first to see if those fields are even necessary.
> 
> For example `served_nh_ids` is used on Inbound Roaming. It indicates other fegs this feg will be forwarding request. Once we have final field on NMS I will add a note to the instructions to use that (right now this is done by the API)
> 
> `served_network_ids` is the ide of the federated LTE networks this FEG serves
> 
> Note both fields are an array, so it would be good to display the format the user needs to use to add more than one

Hey @uri200 , Yes that's right. I understand that user shouldn't configure gateways based on hints. This component is not relevant to the user who is configuring alone, it is also relevant to other read only users. Hints will help them understand the page. We can link the guide if necessary in the config component. cc @jingtien

Reply:
Hey @karthiksubraveti I really don't understand what is the problem. When configuring a feg the person doing so is expected to know that a feg serves an lte network.